lte/gateway/python/magma/subscriberdb/protocols/s6a_proxy_servicer.py

```diff
@@ -128,10 +130,7 @@ def UpdateLocation(self, request, context):
 
             sec_apn.ambr.max_bandwidth_ul = apn.ambr.max_bandwidth_ul
             sec_apn.ambr.max_bandwidth_dl = apn.ambr.max_bandwidth_dl
-            sec_apn.ambr.unit = (
-                s6a_proxy_pb2.UpdateLocationAnswer
-                .AggregatedMaximumBitrate.BitrateUnitsAMBR.BPS
-            )
+            sec_apn.ambr.unit = apn.ambr.br_unit
```

Comment:
double check that this is safe to use. because currently this is not configured anywhere. also you should add unit support on the CLI and s1ap tester to test this.